torch/_torch_docs.py

```diff
@@ -3178,6 +3178,54 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins=10, weights=None, min=0, max=0, density=False) -> Tensor
```

Comment:
"inputs, bins=10, range=None, weights=None, density=False"

range should be a tuple (`float[]?` in native_functions.yaml) like NumPy

Reply:
How am I supposed to do that? I've made the C++ function accept range as an ArrayRef and made it into a float[]? in native_functions.yaml, and now I'm getting an error when trying to build the project.

File "pytorch\tools\pyi\gen_pyi.py", line 154, in type_to_python
    }[typename]
KeyError: 'ArrayRef<double>'

Reply:
Done.

Reply:
Reminder to update it in the docs, too